grpcutils/config.go

```diff
@@ -0,0 +1,8 @@
+package grpcutils
+
+type GrpcAuthenticatorConfig struct {
```

Comment:
```suggestion
type AuthenticatorConfig struct {
```
I think the grpc part can be inferred from that package name

```
grpcutils.GrpcAuthenticatorConfig{}
```
vs 
```
grpcutils.AuthenticatorConfig{}
```

Reply:
We should still rename this one